src/test/java/appModules/Trading_Action.java

```diff
@@ -0,0 +1,260 @@
+		}
+		else if(durationType=="d"){
+			SelectEnterDuration(driver,"0",durationType);
+			Assert.assertEquals(Trade_Page.err_ExpiryTimeTop(driver).getText(), "Expiry time cannot be equal to start time.");
+			Assert.assertEquals(Trade_Page.err_ExpiryTimeBottom(driver).getText(), "Expiry time cannot be equal to start time.");
+			SelectEnterDuration(driver,"366",durationType);
+			Assert.assertEquals(Trade_Page.err_CannotCreateContractTop(driver).getText(), "Cannot create contract");
+			Assert.assertEquals(Trade_Page.err_CannotCreateContractBottom(driver).getText(), "Cannot create contract");
+		}
+	}
+	public static void ValidateAmountField(WebDriver driver,String market,String amount_type){
+		if(market=="Volatility Indices"){
+			SelectEnterAmount(driver,"0.34",amount_type,"5","m");
+			Assert.assertEquals(Trade_Page.err_Payout50000Top(driver).getText(), "Minimum stake of 0.35 and maximum payout of 50,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout50000Bottom(driver).getText(), "Minimum stake of 0.35 and maximum payout of 50,000.00");
+			System.out.println(Trade_Page.err_Payout50000Top(driver).getText());
+			SelectEnterAmount(driver,"51000",amount_type,"5","m");
+			Assert.assertEquals(Trade_Page.err_Payout50000Top(driver).getText(), "Minimum stake of 0.35 and maximum payout of 50,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout50000Bottom(driver).getText(), "Minimum stake of 0.35 and maximum payout of 50,000.00");
+			System.out.println(Trade_Page.err_Payout50000Top(driver).getText());
+		}
+		else if (market=="Forex"){
+			SelectEnterAmount(driver,"0.49",amount_type,"5","m");
+			Assert.assertEquals(Trade_Page.err_Payout5000Top(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout5000Bottom(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			System.out.println(Trade_Page.err_Payout5000Top(driver).getText());
+			SelectEnterAmount(driver,"21000",amount_type,"5","m");
+			Assert.assertEquals(Trade_Page.err_Payout5000Top(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout5000Bottom(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+		}
+		else if (market=="Commodities"){
+			SelectEnterAmount(driver,"0.49",amount_type,"5","m");
+			Assert.assertEquals(Trade_Page.err_Payout5000Top(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout5000Bottom(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			SelectEnterAmount(driver,"5100",amount_type,"5","m");
+			Assert.assertEquals(Trade_Page.err_Payout5000Top(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
+			Assert.assertEquals(Trade_Page.err_Payout5000Bottom(driver).getText(), "Minimum stake of 0.50 and maximum payout of 5,000.00");
```

Comment:
Yes. Seems there were front-end changes to error message in recent release. I've fixed the issue.